### PR TITLE
Foosball scoreboard

### PR DESCRIPTION
**Pull request by reesericci**
## Describe your project in 1-3 sentences

We will be building an IoT scoreboard for the foosball club at our school that will tie into our rating system to automatically update the ratings when a game is completed, and be generally much more fun to use with lights and other things. For example, it will allow for different modes of gameplay like first to 3, 5, and 10, and have the reflected on the scoreboard

## Hack Club Slack handle

@reesericci 

## Checklist before submitting proposal:

- [X] I am a current high school student (or younger).
- [X] I have submitted an RSVP at <https://hackclub.com/winter>
- [X] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [X] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by ItIsTheLucas** [comment]
Hack Club Slack handle

@Lucas Dudley

Checklist before submitting proposal:

 I am a current high school student (or younger).
 I have submitted an RSVP at https://hackclub.com/winter
 I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
 I have self-reviewed my project proposal.

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! this is SOO COOL! **we're back on tuesday and will start reviewing PRs in the order that they came in, thanks for your patience :)**

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts & ready software

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

our Jan 15 deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

File: reesericci.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Foosball Scoreboard
+
+## Summary
+
+We are going to build an IoT scoreboard for the foosball club at our high school that will automatically connect to the rating system we have. It will be basically a strip of NeoPixels for each team and then an arcade button you can hit when you score. We will also have a screen and interface for selecting who's playing and the max point value, to be able to communicate that to the rating system. A stretch goal would be to implement tournaments as well, so it automatically knows who's playing during a tournament.
+
+## Plan
+
+- I will start designing in CAD software the cases for all the different parts like the lights for the scoreboard, the button, screen, and rotary encoders. Then I will start printing them on my 3d printer with the filament. While I'm doing that, Lucas will start wiring things up like the neopixels and screen to the metro, and make a power board so it only needs one power supply.
```
Comment (cfanoulis) [inline reesericci.md:14]: Can you please start assembly during the 10-day building period? If so, please move this to a seperate item (esigning 3D prints & printing is ok)

**Review by cfanoulis — CHANGES_REQUESTED** [review]
hellooooooo! see below + how will you be powering this? dont see any plugs or batteries :/

File: reesericci.md (hunk 1 above)
Comment (reesericci) [reply reesericci.md:14]: They were going to be done concurrently during the 10 days

**Comment by reesericci** [comment]
> how will you be powering this? dont see any plugs or batteries :/

it will be powered over DC - I'll toss that question to @ItIsTheLucas

File: reesericci.md (hunk 1 above)
Comment (reesericci) [reply reesericci.md:14]: I was doing the software CAD stuff and @ItIsTheLucas  assembling things.

**Comment by reesericci** [comment]
@cfanoulis whoops! - I added a DC power supply to the budget.

**Review by cfanoulis — APPROVED** [review]
perfect, tysm for that :) a banker from @hackclub/bank-ops will be in touch to turn your demo project real! enjoy (responsibly)!

**Review by hackliv — APPROVED** [review]
hey @reesericci & @ItIsTheLucas this is so awesome!! I like to think of myself as a top tier foosball player myself, although my sister would probably argue she's better than me. 😆 I'm dming you both on Slack now!

**Pull request merged**